### DISCUSSION TITLE
At least one mod failed to update: Could not find file

### DISCUSSION THREAD

**Discussion by Cherno97** [X2CommunityCore/xcom2-launcher#343] [Help] [answered]
**AML Version: 1.5.0** 

**Description**
for some reason AML is giving me this error since I updated to 1.5.0.

if i'm honest, it never happened to me before, i hope passing these files will help to find the cause.*

**Attachments**
* *Consider attaching the `AML.log` file from the AML folder*
* *Consider attaching the `settings.json` file from the AML folder*
* *If applicable, add screenshots to help explain your problem.*
[AML.log](https://github.com/X2CommunityCore/xcom2-launcher/files/11459809/AML.log)

![0101](https://github.com/X2CommunityCore/xcom2-launcher/assets/133320688/b3e412ba-c699-47d4-b561-71bd903525b7)
 i tried to also attach the settings.json, but github dont let me.

**Comment by RevZero** [comment, marked as answer]
Windows has a file path length limit of 260 chars, but 

`C:\Users\vansa\Downloads\Pa mas adelante\XCOM 2\XCom2-WarOfTheChosen\XComGame\Mods\2608001830_wotc_call_of_duty_modern_warfare_male_operators_outfit_pack_ii\Src\WOTC_COD_MaleOperatorsII_Outfits\Classes\X2DownloadableContentInfo_WOTC_COD_MaleOperatorsII_Outfits.uc`

is 263 chars long. So try renaming 

`2608001830_wotc_call_of_duty_modern_warfare_male_operators_outfit_pack_ii` 
to 
`2608001830_wotc_modern_warfare_male_operators_outfit_pack_ii` 
should suffice.

**Reply by Cherno97** [reply]
Thank you! it was the path length my problem.

Im kinda new in modding so i didnt know of this kind of mistake. Thanks pal!

**Comment by RevZero** [comment]
https://github.com/X2CommunityCore/xcom2-launcher/wiki/Troubleshooting#could-not-find-file
Also moving this to discussions help section.
